### PR TITLE
[IT] Add first support to scenes

### PR DESCRIPTION
**Pull request by deliantwo**
Aggiunto primo supporto alle scene.
Prendendo spunto dall'inglese ho escluso il dominio scene dal turn on generico.
Modificato leggermente la risposta nel caso di una scena.
Aggiunto test relativi, con e senza area.

### CODE REVIEW

File: tests/it/scene_HassTurnOn.yaml
Comment (v1k70rk4) [inline tests/it/scene_HassTurnOn.yaml:1]: Hello, the first line is incorrect, you didn't change the language from 'en' to 'it' there.

Comment (deliantwo) [reply tests/it/scene_HassTurnOn.yaml:1]: Whoops, I've missed that, thank you.
I'm surprised the coherence with the folder is not tested in some way. Maybe I will look into that.

Comment (v1k70rk4) [reply tests/it/scene_HassTurnOn.yaml:1]: Thank you, this is perfect, checking the rest is up to the language leaders :)

Comment (v1k70rk4) [reply tests/it/scene_HassTurnOn.yaml:1]: I don't know how you develop, but if you have the capability to run the lint and test scripts locally from the script library and it checks everything.

Once the language leaders approve the changes, GitHub will also run these and no faulty data will go into the repository.

**Comment by auanasgheps** [comment]
Grazie!

**Pull request merged**